e2e/core/settings-component.e2e.ts

```diff
@@ -0,0 +1,248 @@
+            loginPage.enterPassword(adminUserModel.password);
+            loginPage.clickSignInButton();
+            expect(loginPage.getLoginError()).toMatch(loginError);
+        });
+    });
+
+    describe('Settings Component - BPM Basic Authentication', () => {
+        beforeAll( (done) => {
+            settingsPage.goToSettingsPage();
+            settingsPage.setProvider(settingsPage.getEcmAndBpmOption(), 'ALL');
+            settingsPage.setContentServicesURL(TestConfig.adf.url);
+            settingsPage.setProcessServicesURL(TestConfig.adf.url);
+            settingsPage.clickApply();
+            done();
+        });
+
+        it('[C277751] Should allow the User to login to Process Services using the BPM selection on Settings page', () => {
+            loginPage.goToLoginPage();
+            loginPage.clickSettingsIcon();
+            settingsPage.checkProviderDropdownIsDisplayed();
+            settingsPage.checkProviderOptions();
+            settingsPage.checkBasicAuthRadioIsSelected();
+            settingsPage.checkSsoRadioIsNotSelected();
+            expect(settingsPage.getEcmHostUrl()).toBe(TestConfig.adf.url);
+            expect(settingsPage.getBpmHostUrl()).toBe(TestConfig.adf.url);
+            expect(settingsPage.getBackButton().isEnabled()).toBe(true);
+            expect(settingsPage.getApplyButton().isEnabled()).toBe(true);
+            loginPage.goToLoginPage();
+            loginPage.clickSettingsIcon();
+            settingsPage.checkProviderDropdownIsDisplayed();
+            settingsPage.setProvider(settingsPage.getBpmOption(), 'BPM');
+            settingsPage.clickBackButton();
+            loginPage.waitForElements();
+            loginPage.clickSettingsIcon();
+            settingsPage.checkProviderDropdownIsDisplayed();
+            settingsPage.setProviderBpm();
+            loginPage.waitForElements();
+            loginPage.enterUsername(adminUserModel.id);
+            loginPage.enterPassword(adminUserModel.password);
+            loginPage.clickSignInButton();
+            navigationBarPage.navigateToProcessServicesPage();
+            browser.getCurrentUrl().then((url) => {
+                expect(url).toBe(TestConfig.adf.url + '/activiti');
+            });
+            navigationBarPage.navigateToSettingsPage();
+            expect(settingsPage.getSelectedOptionText()).toBe('BPM');
+            settingsPage.checkBasicAuthRadioIsSelected();
+            settingsPage.checkSsoRadioIsNotSelected();
+            expect(settingsPage.getBpmHostUrl()).toBe(TestConfig.adf.url);
+            expect(settingsPage.getBackButton().isEnabled()).toBe(true);
+            expect(settingsPage.getApplyButton().isEnabled()).toBe(true);
+            settingsPage.clickBackButton();
+            loginPage.waitForElements();
+            browser.get(TestConfig.adf.url + '/activiti');
+            browser.getCurrentUrl().then((url) => {
+                expect(url).toBe(TestConfig.adf.url + '/activiti');
+            });
```

Comment:
It might be better, instead of checking the URL, checking if an element of that page is displayed.

Reply:
Is there an element that can be checked here? and will be available across any environment we test. Please let me know.

Reply:
For Process Services page you should always have Task App and for Content Services page there's the document list.

Reply:
thanks, updated the test.